<launch>
  <node name="map_co2" pkg="gradient_map_server" type="map_server" args="$(find woz_sensor_maps)/3rdfloor_10cm_co2.yaml">
    <remap from="map" to="/sensor_maps/co2"/>
    <remap from="static_map" to="/map_co2/static_map"/>
  </node>
</launch>
